### PR TITLE
exclude `showNA` while checking for changes in attribute from default state

### PR DESCRIPTION
**Pull request by kalletlak**

### CODE REVIEW

File: src/pages/studyView/StudyViewPageStore.ts [hunk 1]
```diff
@@ -3799,7 +3799,10 @@ export class StudyViewPageStore {
                     }
                     const attributeChanged =
                         isDefaultAttr &&
-                        !_.isEqual(toJS(attribute), initDataBinFilter);
+                        !_.isEqual(
```
Comment (alisman) [inline src/pages/studyView/StudyViewPageStore.ts:3802]: we can probably just merge this, but would it not be better to test for what we actually want to compare?

Comment (alisman) [reply src/pages/studyView/StudyViewPageStore.ts:3802]: as in, what properties of the attribute need to be different in order for this to be regarded as changed

Comment (kalletlak) [reply src/pages/studyView/StudyViewPageStore.ts:3802]: we need to compare for everything except `showNA`

File: src/pages/studyView/StudyViewPageStore.ts [hunk 2]
```diff
@@ -6355,7 +6364,7 @@ export class StudyViewPageStore {
     }
 
     @computed private get hasSampleIdentifiersInFilter(): boolean {
-        return !(
+        return (
```
Comment (alisman) [inline src/pages/studyView/StudyViewPageStore.ts:6367]: this seems weird

**Review by adamabeshouse — APPROVED** [review]

**Comment by inodb** [comment]
@kalletlak could you help update the description to clarify what this change does? This avoids the duplicate binning related HTTP request on study view?

**Comment by kalletlak** [comment]
@inodb yes

**Pull request merged**